package.json

```diff
@@ -4,11 +4,15 @@
     "start:dev": "snowpack dev",
     "start:firestore": "firebase emulators:start --import=./devData --only firestore",
     "build": "snowpack build",
-    "test": "jest --watch --FIRESTORE_EMULATOR_HOST=8080",
+    "test": "jest --watch",
     "test:ci": "jest",
+    "test:e2e": "start-server-and-test start:dev 3000 cy:run",
+    "test:e": "run-p start:dev cy:open",
```

Comment:
You've also ripped out the firestore emulator here as well - meaning you are telling it to always hit our live DB.

I will probably make a PR to clean up these scripts a bit, as they are getting out of hand :stuck_out_tongue: